src/explore-education-statistics-admin/src/pages/release/data/components/ApiDataSetFinaliseBanner.tsx

```diff
@@ -6,12 +6,14 @@ import Button from '@common/components/Button';
 import React from 'react';
 
 interface Props {
+  changelogPath: string;
```

Comment:
Why make this a configurable prop? The link can just be hard-coded to be generated from `releaseApiDataSetChangelogRoute`?

Reply:
Good question... no idea why I did that, I've changed it now.